config.py

```diff
@@ -111,3 +111,10 @@
     'LabHub:*': {'allowprivate': False}}
 
 AUTOINSTALL_DEPS = True
+
+DEFAULT_CONFIG = {
+    'SpammingAlert': {
```

Comment:
Please do not put any overrides here.

These are the same as the bot defaults, so unnecessary.